rescript/plugin_setup.py

```diff
@@ -828,6 +830,34 @@
         "used for slicing the original alignment."),
 )
 
+T = TypeMatch([AlignedSequence, Sequence])
+plugin.methods.register_function(
+    function=subsample_fasta,
+    inputs={'sequences': FeatureData[T], },
```

Comment:
no need for the trailing comma

Reply:
But it `flake`s!

Reply:
Oh, wait, it actually doesn't - sorry 🤦  Will clean up!